core/src/com/unciv/ui/worldscreen/WorldScreen.kt

```diff
@@ -175,8 +181,68 @@ class WorldScreen(val gameInfo: GameInfo, val viewingCiv:CivilizationInfo) : Cam
     }
 
     private fun addKeyboardPresses() {
+        val quickSave = {
+            val toast = ToastPopup("Quicksaving...", this)
+            thread(name = "SaveGame") {
+                GameSaver.saveGame(gameInfo, "QuickSave") {
+                    Gdx.app.postRunnable {
+                        toast.close()
+                        if (it != null)
+                            ToastPopup("Quicksave failed!", this)
+                        else {
+                            ToastPopup("Quicksave OK.", this)
+                            UncivGame.Current.setWorldScreen()
```

Comment:
Why is this necessary? We're already on the world screen, and it won't change because of a quicksave.

Reply:
... copied without thinking too much? As I said, there's already two differently structured copies out there on how the `GameSaver.saveGame` call is wrapped, one for the save screen, one for autosaves (not counting multiplayer) if I remember correctly, and once that code is unified we'd simply call it. Question is - who has the will to attack this monster task...